src/pages/accessManagment/login/view.js

```diff
@@ -25,7 +25,10 @@ function getRedirectPath(query) {
 
 function loginView({ classes }) {
   const { signInWithGoogle, signInWithNIH } = useAuth();
-  const { authProviders } = globalData;
+  let { authProviders } = globalData;
```

Comment:
Looks like empty authProviders situation is handled below for this page.

Reply:
Line 54.

Reply:
Yes, that's duplicate will fix it soon.